### ISSUE TITLE
Richer states for event (to be sent, sending ...)

### ISSUE THREAD

**Issue by IpsmLorem** [wireapp/wire-ios#198]
It would be interesting to have richer state for event: 
1. to be sent: the message in on the sending stack. cf. #197
2. sending: the mesage is travelling through network and is not finished to be transfered
3. sent: the message is on the Wire server and left the client
4. delivered: for each recipient, the message was sent to at least one recipient device
5. read: for each recipient, the message was read by at least one device

The 4 and 5 states could be optionnal according to the recipient preferences.

**Comment by david-wire** [comment]
Hello,

Thanks for the feedback. We are always looking for ways to improve Wire. 
We have implemented 2. where you can see if the message is being sent, 3. as well (if you tap on the actual message). 4. working on it.
As for 5. we are not planning on implementing this, for privacy reasons.

I hope that answers your question.

**Comment by IpsmLorem** [comment]
Thank you for the reply.

> As for 5. we are not planning on implementing this, for privacy reasons.
Maybe a turned off by default option ? (secure by default policy)

4 is more important than 5 and does not serve the same purpose anyway.

**Comment by marcoconti83** [comment]
Hi @IpsmLorem . 
We introduced "delivered" status for one to one conversations. It's already in the latest App Store version. Try it out!
We do not have plans to introduce it for group conversations, as it makes less sense there and it introduces a series of technical problems.
We considered adding the "read" status but we decided not to go for it for the moment because of the privacy implications.
